lola.html

```diff
@@ -670,19 +664,92 @@ <h4>Copying “replies” and “inReplyTo” </h4>
       <h4>Adding Breadcrumbs</h4>
       
       <p>
-        As the destination server copies Activity and Like objects, it SHOULD add breadcrumbs.  It SHOULD preserve the existing breadcrumbs because it’s nice to do so.  Breadcrumbs should be pushed on the top (or beginning) of a most-recently-first list in the ‘previously’ attribute.
+        As the destination server copies Activity and Like objects, it SHOULD add breadcrumbs.  It SHOULD preserve the existing breadcrumbs because it’s nice to do so.  Breadcrumbs should be pushed on the top (or beginning) of a most-recently-first list in the ‘previously’ property.  The 'previously' attribute is defined in this specification as a multi-valued property, as follows: 
+      </p>
+
+      <table>
+        <tr><td>URI:</td><td>TBD</td></tr>
+        <tr><td>Notes:</td><td>One or more actor/ID pairs</td></tr>
+        <tr><td>Domain:</td><td>Activity</td></tr>
+        <tr><td>Range:</td><td>Object | Link</td></tr>
+      </table>
+
+      <p>Example:</p>
+      <pre>
+"type": "Article",
+"id": "https://thirdhome.example/meadowvine/posts/xyz",
+"previously": [
+  { 
+    "actor": "https://newsite.example.org/aurora/",
+    "id": "https://newsite.example.org/aurora/items/02751cab-..."
+  },
+  { 
+    "actor": "https://lemongrove.example.co.uk",
+    "id": "https://lemongrove.example.co.uk/2016/05/minimal-activitypub"
+  }
+]
+      </pre>
+      <p>The "actor" value SHOULD be included, because there is no guarantee that a 3rd party can derive the actor value from 
+        an "id" value. The "actor" value is needed in order to validate the previous "id" value before trusting it.  Because
+        the destination server simply states the "previously" value, it cannot be entirely trusted.  A 3rd party MAY check the 
+        original actor to make sure that it does redirect to the new Actor, but that check is not guaranteed to work because the
+        source server hosting the original Actor may be inactive or have removed that Actor.  
       </p>
 
+      <p>
+        If an object lists an Actor/id in "previously", and that actor does not have a movedTo value, or if that movedTo
+        value does not link (possibly in several hops) to the object's current location, then that "previously" value cannot 
+        be trusted.  If a "previously" value cannot be trusted, the expected behavior (though not REQUIRED) is that it is ignored. 
+        Ignoring a "previously" value reduces the situation to an existing common situation, where ActivityPub agents 
+        cannot be sure if an Object had been copied from somewhere else or not.  ActivityPub agents MAY keep a cache of 
+        moved Actor history if they want to be more efficient in resolving and relying on breadcrumbs, although note that
+        a cache that is not carefully designed could prevent a user from recovering from an ill-destined move, by
```

Comment:
```suggestion
        a cache that is not carefully designed could prevent a user from recovering from an infelicitous or corrupted move operation, by
```
(or go headlong in the other direction and go with `star-crossed` 🎭  )